faq.md

```diff
@@ -52,13 +52,7 @@ presentation and semantics.
 
 **How is the MathML association different from the MathJax Consortium?**
 
-As indicated on the MathJax website, "The core of the MathJax project is the
-development of its state-of-the-art, open source, JavaScript platform for
-display of mathematics.". Instead, the MathML association does not focus
-on promoting a particular product but intends to implement the MathML standard
-into Web browsers and assistive technologies. Note that MathJax's internal
-representation is MathML and its team members have advocated for native MathML
-support in Web browsers, but they have never made any official statement
-regarding funding plans. Moreover, the way MathJax encapsulates MathML makes it
-badly integrated into the Web platform and hardly accessible to MathML-aware
-tools ; hence this can only be a transition technology.
+We are happy that MathJax has already embraced MathML, which is used as its core formula representation language. While MathJax has provided an invaluable bridge solution for displaying beautiful math-on-the-web today, the MathML association is focusing on advancing the state of web mathematics tomorrow.
+
+Our goal towards standards-compliant web mathematics, via native MathML support in all browsers, aims at faster rendering, better accessibility and towards building a healthy ecosystem of math web application development. As such, our goals are potentially complementary to the MathJax Consortium and we are open to collaborate with them to this end.
+
```

Comment:
I think "collaborate"  and "potentially complementary" are still vague and I don't know if we are really able to claim the latter given that even the MathJax Consortium is ambiguous about whether they want native MathML. So I'd prefer my previous proposal that clearly state our position (reference to the manifesto) and what we know about MathJax's one (quotation from their website and public statement), that opens participation via our clearly-defined affiliate status and that lets them the opportunity to clarify their position.